_haikus/2022_hacktorberfest_myfirstpr.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: My First PR
+author: Vedangi Bhujang
+---
+
+This is my first PR.<br>
```

Comment:
```suggestion
It's my first PR.<br>
```

Reply:
@ltagliaferri  Hey, thanks for the suggestions. I have made the changes.